app/assets/javascripts/controllers/ops/pglogical_replication_form_controller.js

```diff
@@ -149,6 +149,11 @@ ManageIQ.angular.app.controller('pglogicalReplicationFormController', ['$http',
     }
     $scope.pglogicalReplicationModel.addEnabled = false;
     $scope.pglogicalReplicationModel.updateEnabled = false;
+
+    // check if subscriptions changed
+    if (angular.equals($scope.pglogicalReplicationModel.subscriptions, $scope.modelCopy.subscriptions)) {
```

Comment:
You may want to check the whole model here.

When you change another field and then add a subscription identical to what was in model copy, it would mark the whole form as pristine.

Reply:
OK, in this case, the only other thing in the model is the type,
changing type to global (and saving) clears all existing subscriptions (https://github.com/ManageIQ/manageiq/blob/master/app/models/miq_region.rb#L163-L172),
and having it set to remote without adding any subscriptions doesn't make sense,
so this can't be a problem in practice :+1: 

Thanks for making sure! :)

Reply:
I would say that if we are a global and are changing to something else (none or remote) we delete all the subscriptions: https://github.com/ManageIQ/manageiq/blob/master/app/models/miq_region.rb#L168 so then later if we change from Remote to Global type, all the original Subscriptions will be already deleted.